### PR TITLE
style: global font size change to 14px

### PR DESCRIPTION
**Pull request by shikharmohan**
Adding new mixins for upcoming font changes

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @shikharmohan has signed the CLA. Thanks for the pull request!

File: src/styles/common/_typography.scss [hunk 1]
```diff
@@ -19,6 +19,10 @@ body {
     @include common-typography;
 }
 
+.Body.is-deja-blue {
```
Comment (alexkrolick) [inline src/styles/common/_typography.scss]: Check that this also affects modals and other detached roots

Comment (shikharmohan) [reply src/styles/common/_typography.scss]: modals/portalled elements have this new font size

File: src/styles/mixins/_typography.scss [hunk 2]
```diff
@@ -19,10 +19,26 @@
...
     color: $bdl-gray;
     line-height: $bdl-line-height;
     letter-spacing: .3px;
 }
+
+@mixin common-typography-deja-blue {
+    @include fonts-deja-blue;
+
+    color: $bdl-gray;
+    line-height: $bdl-line-height;
```
Comment (tjgupta) [inline src/styles/mixins/_typography.scss]: Is the line height supposed to be the same with the larger font?

Comment (shikharmohan) [reply src/styles/mixins/_typography.scss]: same line height: 20px

File: src/styles/common/_typography.scss [hunk 3]
```diff
@@ -19,6 +19,10 @@ body {
     @include common-typography;
 }
 
+body.is-deja-blue {
```
Comment (shikharmohan) [inline src/styles/common/_typography.scss]: Not going to change this top level class, requires monolith changes. Considering it out of scope right now.

File: src/styles/constants/_typography.scss [hunk 4]
```diff
@@ -1 +1,5 @@
 $bdl-line-height: 20px;
+
+// font sizes
+$bdl-font-size: 13px;
+$bdl-font-size--dejablue: 14px;
```
Comment (priyajeet) [inline src/styles/constants/_typography.scss]: fontSize, likewise camel casing other places too

File: src/styles/mixins/_typography.scss (hunk 2 above)
Comment (priyajeet) [reply src/styles/mixins/_typography.scss]: I would prefix all these with bdl-

bdl-typography
bdl-commonTypography

File: src/styles/constants/_typography.scss (hunk 4 above)
Comment (alexkrolick) [reply src/styles/constants/_typography.scss]: None of the other variables are like that

Comment (shikharmohan) [reply src/styles/constants/_typography.scss]: I'm cool w/ whatever we go with, lmk!

File: src/styles/mixins/_typography.scss [hunk 5]
```diff
@@ -16,7 +16,15 @@
     @include font-aliasing;
 
     font-weight: normal;
-    font-size: 13px;
+    font-size: $bdl-fontSize;
+}
+
+@mixin bdl-fonts-dejaBlue {
```
Comment (priyajeet) [inline src/styles/mixins/_typography.scss]: --dejaBlue if you want to keep it as a modifier pattern like you did with the variable name

**Pull request merged**